test/Microsoft.AspNetCore.AzureAppServices.FunctionalTests/TemplateFunctionalTests.cs

```diff
@@ -34,6 +34,91 @@ public TemplateFunctionalTests(AzureFixture fixture, ITestOutputHelper outputHel
         }
 
         [Theory]
+        [InlineData("1.0.5", "web", "Hello World!")]
+        [InlineData("1.0.5", "mvc", "Learn how to build ASP.NET apps that can run anywhere")]
+        [InlineData("1.1.2", "web", "Hello World!")]
+        [InlineData("1.1.2", "mvc", "Learn how to build ASP.NET apps that can run anywhere")]
+        public async Task LegacyTemplateRuns(string expectedRuntime, string template, string expected)
+        {
+            var testId = nameof(LegacyTemplateRuns) + template + expectedRuntime.Replace(".", string.Empty);
+
+            using (var logger = GetLogger(testId))
+            {
+                var site = await _fixture.Deploy("Templates\\AppServicesWithSiteExtensions.json",
+                    baseName: testId,
+                    additionalArguments: new Dictionary<string, string>
+                    {
+                        { "extensionFeed", AzureFixture.GetRequiredEnvironmentVariable("SiteExtensionFeed") },
+                        { "extensionName", "AspNetCoreTestBundle" },
+                        { "extensionVersion", GetAssemblyInformationalVersion() },
+                    });
+
+                var testDirectory = GetTestDirectory(testId);
+
+                var dotnet = DotNet(logger, testDirectory, "1.1");
+
+                await dotnet.ExecuteAndAssertAsync("new " + template);
+
+                UpdateCSProj(testDirectory, $"Templates\\Legacy.{expectedRuntime}.{template}.csproj");
+
+                InjectMiddlware(testDirectory, RuntimeInformationMiddlewareType, RuntimeInformationMiddlewareFile);
+
+                var pp = await site.GetPublishingProfileAsync();
+
+                var git = Git(logger, testDirectory);
+                await git.ExecuteAndAssertAsync("init");
+                await git.ExecuteAndAssertAsync($"remote add origin https://{pp.GitUsername}:{pp.GitPassword}@{pp.GitUrl}");
+                await git.ExecuteAndAssertAsync("add .");
+                await git.ExecuteAndAssertAsync("commit -am Initial");
```

Comment:
This needs quotes around the message right?

Reply:
One word message